content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md

```diff
@@ -496,7 +496,7 @@ topology spread constraints are applied to a Pod if, and only if:
 
 Default constraints can be set as part of the `PodTopologySpread` plugin
 arguments in a [scheduling profile](/docs/reference/scheduling/config/#profiles).
-The constraints are specified with the same [API above](#api), except that
+The constraints are specified with the same [API above](#topologyspreadconstraints-field), except that
```

Comment:
nit: Could you provide more descriptive link text instead of `API above`? We recommend to avoid using 'below', 'above', 'here' to describe a link.